app/core/constants.js

```diff
@@ -36,6 +36,7 @@ export const DATA_DONATION_NONPROFITS = [
   { value: 'JDRF', label: 'JDRF' },
   { value: 'NSF', label: 'Nightscout Foundation' },
   { value: 'T1DX', label: 'T1D Exchange' },
+  { value: 'AADE', label: 'American Association of Diabetes Educators Foundation' },
```

Comment:
What I meant in the previous change request was that we want the label to be `AADE Foundation` exactly (i.e. not expanding the `AADE` abbreviation).